src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -913,6 +913,7 @@ public virtual VersionControlDetails VisitVersionControlDetails(VersionControlDe
         {
             if (node != null)
             {
+                node.MappedTo = VisitNullChecked(node.MappedTo);
```

Comment:
>MappedTo [](start = 21, length = 8)

Note that `rule.deprecatedIds` does _not_ induce a change in the visitor because it's an array of a primitive type.